README.md.jinja

```diff
@@ -0,0 +1,59 @@
+# Frequenz {{project_name | replace("-", " ")}}
+
+{{project_description}}
+
+{% if python %}
+## Python Bindings
+
+To work with the Python bindings, first activate a Python Environment in which
+you want to run/build your Rust features.
+
+### Building
+
+To build the wheels, run:
+
+```bash
+pip install -r requirements.txt
+maturin build --release --features python -o wheels -i $$(which python)
+```
+
+This will install the required dependencies from
+[requirements.txt](./requirements.txt), build the wheel,
+and store it in the [./wheels](./wheels) directory.
+
+### Installing
+
+To install the project directly to the currently active Python environment, run:
+
+```bash
+pip install --force-reinstall -U ./wheels/frequenz_{{project_name | replace("-", "_")}}-*.whl
+```
+
+This will build the wheel and then install it using `pip`.
+
+### Deploying
+
+To deploy the wheels to a PyPI repository, run:
+
+```bash
+maturin upload ./wheels/frequenz_{{project_name | replace("-", "_")}}-*.whl
+```
+
+This will upload the wheels in the [./wheels](./wheels) directory to the PyPI
+repository.
```

Comment:
Again, I wouldn't encourage this way of deploying, I guess `maturin` would still be useful for **bulding**, but for **uploading** we should use [trusted publishers](https://docs.pypi.org/trusted-publishers/) via GitHub Actions.

Reply:
okay, I see. Let me remove it.

Reply:
Thanks. I think we can merge this as is to move forward.

In a follow-up PR you could add a release workflow that publishes to pypi (and maybe creates the GitHub release too) as [we have in `repo-config`](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/3bd90e8508940f7f6fe705aa9af0fd3b8e45a9c1/cookiecutter/%7B%7Bcookiecutter.github_repo_name%7D%7D/.github/workflows/ci.yaml#L464-L529) and add a section about deployment back (you can use [`repo-config` as inspiration](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/cookiecutter/%7B%7Bcookiecutter.github_repo_name%7D%7D/CONTRIBUTING.md#releasing) too).